lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -510,11 +510,17 @@ defmodule ExUnit.Case do
 
       defmodule MyTest do
         use ExUnit.Case
-        ExUnit.Case.register_attribute __ENV__, :foobar
 
-        @foobar hello: "world"
-        test "using custom test attribute", context do
-          assert context.registered.hello == "world"
+        ExUnit.Case.register_attribute __ENV__, :fixtures, accumulate: true
```

Comment:
@josevalim Does it make sense that this function accepts both env or module? I think it could be simplified to accept only modules, like `Module.register_attribute/3`.

@edwinthinks I would use `__MODULE__` instead of `__ENV__`.